examples/src/main/python/mllib/fpgrowth_example.py

```diff
@@ -15,10 +15,10 @@
 # limitations under the License.
 #
 
-# $example on$
-from pyspark.mllib.fpm import FPGrowth
 # $example off$
 from pyspark import SparkContext
+# $example on$
```

Comment:
This rearrangement of the example text is going to be weird (you now have the off directive before the on directive).